synce_ql_manager.h

```diff
@@ -0,0 +1,22 @@
+/**
+ * @file synce_ql_manager.h
+ * @brief Interface for managing QL for external clock sources
+ * @note SPDX-FileCopyrightText: Copyright 2022 Intel Corporation
+ * @note SPDX-License-Identifier: GPL-2.0+
+ */
+#ifndef HAVE_SYNCE_QL_MANAGER_H
+#define HAVE_SYNCE_QL_MANAGER_H
+
+#define PORT 			12345
```

Comment:
the port shall be configurable I guess

Reply:
in ptp4l it is hard coded. I think we just need to understand what to use

Reply:
port no longer there